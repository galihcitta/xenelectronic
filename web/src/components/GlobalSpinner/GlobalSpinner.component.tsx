import React from 'react';

import { SpinnerContainer, SpinnerOverlay } from './GlobalSpinner.styles';

export const GlobalSpinner: React.FC = () => (
  <SpinnerOverlay data-testid="Spinner">
    <SpinnerContainer />
  </SpinnerOverlay>
);
