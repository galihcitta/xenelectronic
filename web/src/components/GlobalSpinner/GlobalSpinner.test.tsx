import React from 'react';
import { render } from '@testing-library/react';
import { GlobalSpinner } from './GlobalSpinner.component';

describe('<GlobalSpinner />', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<GlobalSpinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
