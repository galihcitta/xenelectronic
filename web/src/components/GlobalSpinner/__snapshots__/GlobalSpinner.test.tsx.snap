// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalSpinner /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa ijHJlI"
    data-testid="Spinner"
  >
    <div
      class="sc-bwzfXH cdCsLX"
    />
  </div>
</DocumentFragment>
`;
